lldb/test/API/lang/swift/nsarray_code_running_formatter/main.swift

```diff
@@ -23,7 +23,7 @@ func main() {
   var tb = Test() as Array + []
 
   print("second stop") //% self.expect('frame variable -d run -- t', substrs=['t = 0x', 'NSArray = {', 'NSObject = {'])
-                       //% self.expect('frame variable -d run -- ta', substrs=['ta = {', '_buffer = {', '_storage =', 'rawValue = 0x'])
+                       //% self.expect('frame variable -d run -- ta', substrs=['ta = <uninitialized>', '_buffer = {', '_storage =', 'rawValue = 0x'])
```

Comment:
Note that this test was basically showing that the formatter didn't work here before either.